### PR TITLE
This will resolve the ios stuck issue

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
1) Instead of dialog show info directly on the screen.
2) Add try again and other action buttons for each state.

### CODE REVIEW

File: lib/screens/routing_screen.dart [hunk 1]
```diff
@@ -32,114 +47,175 @@ class _RoutingScreenState extends State<RoutingScreen> {
...
-                      valueListenable: username,
-                      builder: (_, String name, __) {
-                        return Text("$kWelcomeToEaselText $name");
-                      }),
-                ],
-              ),
+    Widget child = const SizedBox();
+    switch (state) {
+      case RoutingScreenState.initial:
+        child = const Text("Please wait");
```
Comment (kjawadDeveloper2) [inline lib/screens/routing_screen.dart]: Add in constant file.

File: lib/screens/routing_screen.dart [hunk 2]
```diff
@@ -32,114 +47,175 @@ class _RoutingScreenState extends State<RoutingScreen> {
...
+            style: TextStyle(fontSize: 20.sp),
+          ),
+          SizedBox(
+            height: 50.h,
+          ),
+          PylonsButton(
+            onPressed: () async {
+              onDownloadNowPressed(context);
+            },
+            btnText: "Install App",
```
Comment (kjawadDeveloper2) [inline lib/screens/routing_screen.dart]: Add in constant file.

File: lib/screens/routing_screen.dart [hunk 3]
```diff
@@ -32,114 +47,175 @@ class _RoutingScreenState extends State<RoutingScreen> {
...
-              PylonsWallet.instance.goToPylons();
+  Container buildAccountNotCreated(BuildContext context) {
+    return Container(
+      padding: EdgeInsets.symmetric(horizontal: 40.w),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          Text(
+            "You don't have Pylons account",
```
Comment (kjawadDeveloper2) [inline lib/screens/routing_screen.dart]: Add in constant file.

File: lib/screens/routing_screen.dart [hunk 4]
```diff
@@ -32,114 +47,175 @@ class _RoutingScreenState extends State<RoutingScreen> {
...
-              );
+  Container buildSomethingWentWrong(BuildContext context) {
+    return Container(
+      padding: EdgeInsets.symmetric(horizontal: 40.w),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        mainAxisSize: MainAxisSize.min,
+        children: [
+          Text(
+            "Something went wrong. Try again later",
```
Comment (kjawadDeveloper2) [inline lib/screens/routing_screen.dart]: Add this in constant file.

**Review by mustafapylons — APPROVED** [review]
Looks good to me.

File: lib/screens/routing_screen.dart [hunk 5]
```diff
@@ -1,145 +1,215 @@
-import 'package:easel_flutter/easel_provider.dart';
```
Comment (afti-githobo) [inline lib/screens/routing_screen.dart:1]: shouldn't this whole file just be deleted?

**Review by afti-githobo — CHANGES_REQUESTED** [review]

File: lib/screens/routing_screen.dart (hunk 5 above)
Comment (kjawadDeveloper2) [reply lib/screens/routing_screen.dart:1]: Done

**Review by afti-githobo — APPROVED** [review]

**Review by kiroj305 — APPROVED** [review]

**Pull request merged**